README.md

```diff
@@ -17,18 +17,46 @@
 #
 -->
 
-# apisix-control-plane
+# APISIX-CONTROL-PLANE
 
-## what is apisix-control-plane?
-apisix-control-plane provide APISIX with a `yml` configuration capability,
-We can use `yml` to define the proxy behavior of APISIX
+## What is apisix-control-plane
 
-## Why do you want to do this?
-1. In order to facilitate the integration of k8s, use `yml` to define APISIX;
-2. For easier synchronization across clusters;
-3. Can be better adapted to multiple platforms (k8s, vm);
+apisix-control-plane is an implementation of providing a control plane for apache apisix. with a `yml` configuration capability, We can use `yml` to define the behavior of APISIX.
```

Comment:
We not just want to support APISIX, and will support other DP

Reply:
I wrote [here](https://github.com/apache/apisix-control-plane/pull/27/files#diff-04c6e90faac2675aa89e2176d2eec7d8R46) that can provide configuration capabilities for other gateways.
Let me add another explanation